pepper-apis/src/main/java/org/broadinstitute/ddp/filter/StudyAdminAuthFilter.java

```diff
@@ -21,6 +21,10 @@
 
     @Override
     public void handle(Request request, Response response) {
+        if (request.requestMethod().equals("OPTIONS")) {
+            return;
+        }
+
```

Comment:
Same as what `UserAuthCheckFilter` is doing. (https://github.com/broadinstitute/ddp-study-server/blob/develop/pepper-apis/src/main/java/org/broadinstitute/ddp/filter/UserAuthCheckFilter.java#L57)